src/docs/releases/1.9.0.md

```diff
@@ -22,6 +22,13 @@ Introducing a new "Azure AI Search" module, designed to empower you in the admin
 
 Added new extensions to make registering custom deployment step easier
 
- - services.AddDeployment<TSource, TStep>();
- - services.AddDeployment<TSource, TStep, TDisplayDriver>();
- - services.AddDeploymentWithoutSource<TStep, TDisplayDriver>();
+- services.AddDeployment<TSource, TStep>();
+- services.AddDeployment<TSource, TStep, TDisplayDriver>();
+- services.AddDeploymentWithoutSource<TStep, TDisplayDriver>();
+
+## Users Module
+
+Added new localization settings that allow the user to set his/her default culture and time zone.
+
+!!! note
+    The newly added settings override the current admin localization culture and time zone settings.
```

Comment:
```suggestion
Added new localization settings in the User Localization feature that allow the user to set their default culture and time zone.

!!! note
    The newly added settings override the current admin localization culture and time zone settings if the User Localization feature is enabled.
```

Reply:
Feel free to commit these directly and let the conversations auto-resolve themselves.